src/main/java/com/hbm/blocks/network/FluidDuctStandard.java

```diff
@@ -70,7 +75,42 @@ public void getSubBlocks(Item item, CreativeTabs tab, List list) {
 	public int damageDropped(int meta) {
 		return rectify(meta);
 	}
+	/*
```

Comment:
That is a lot of commented code, what happened here?

Reply:
The horrible attempt at the creative convenience, you can call it “mk1”


also i got rushed and i really wanted to pr this i am sorry bob